.github/workflows/tag-pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Tag release
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
```

Comment:
I think we should keep the tagging process as manual. That would be our trigger to cut a release. Looking at other open-source crates, they don't typically publish every commit. Plus, we'd like to be able to choose the appropriate version bump level (patch, minor, major).

Reply:
Sure. I will remove the automated workflow for tagging.